### ISSUE TITLE
Email service config to support disabling of TLS

### ISSUE THREAD

**Issue by seeya** [pennersr/shove#8]
Hi, I would very like to use this project as my push notification service in production.
I'm currently using the email service. However due to some network restriction from my department, I am only able to send email without TLS. 

I've looked through the implementation and saw the the email service uses ` https://github.com/jordan-wright/email` package.
As you've already encoded the data nicely in `body` I can recreate the `*Email` object and send it using the email package.
With the extra config at the end `&tls.Config{InsecureSkipVerify: true}` I was able to send the email out successfully.

It would be great if we could include additional settings to configure the email client when sending out the email. 
I added the snippet which I am using now. Do let me know if it make sense and whether I can help to contribute to the core codes. 

```golang
func (ec EmailConfig) send(from string, to []string, body []byte, fc services.FeedbackCollector) error {
        t := time.Now()
        email, _ := jwemail.NewEmailFromReader(bytes.NewReader(body))
        err := email.SendWithStartTLS(fmt.Sprintf("%s:%d", ec.EmailHost, ec.EmailPort), nil, &tls.Config{InsecureSkipVerify: true})
        duration := time.Since(t)
        fc.CountPush(serviceID, err == nil, duration)

        if err != nil {
                ec.Log.Printf("[ERROR] Send failed: %s", err)
                return err
        }
        return nil
}
```

**Comment by pennersr** [comment]
I would prefer not to use that `NewEmailFromReader` method, because I think that the underlying code has a simplified view of an email and is possibly unable to handle complex digested emails consisting of multiple parts. Also, it may be worthwhile to swap that package out in favour of another one, so I think it is good to introduce a local method capable of sending mails by TLS.

Can you have a look at commit c3c814fb2be82048de42888155bb7bd78d260435 in branch `feat-email-tls` -- does that cover your needs?

**Comment by seeya** [comment]
Hi, thanks for creating the branch. I've tested the code and it works if the "from" value is just an email. I think the `Subject` is also missing from the email.

## Doersn't work with from value "Test <noreply@test.com>"
```json
{
    "digest": {
        "subject": "Hello Digest"
    },
    "subject": "TEST SUBJECT",
    "from": "Test <noreply@test.com>",
    "to": ["recipeint@test.com"],
    "text": "This is a text email",
    "html": "<!DOCTYPE html>\n<html=\"en\">\n<body>\n<p><b>This is a html message FROM GOLANG!</b></p>\n</body>"

}
```

## Works
```json
{
    "digest": {
        "subject": "Hello Digest"
    },
    "subject": "TEST SUBJECT",
    "from": "noreply@test.com",
    "to": ["recipeint@test.com"],
    "text": "This is a text email",
    "html": "<!DOCTYPE html>\n<html=\"en\">\n<body>\n<p><b>This is a html message FROM GOLANG!</b></p>\n</body>"
}
```

**Comment by pennersr** [comment]
I doubt there is an issue with respect to the subject -- this service is in production and subjects work just fine over there. As for the from, I'll have a look...

**Comment by pennersr** [comment]
The TLS support has landed on master, and this should fix from encoding: cd866cc

**Comment by seeya** [comment]
Appreciate the TLS support. Tested on my side and it works flawlessly!
